numba/tests/test_extending.py

```diff
@@ -709,5 +716,16 @@ def bar(n):
         self.assertIn(msg, str(raises.exception))
 
 
+class TestImportCythonFunction(unittest.TestCase):
+    @unittest.skipIf(sc is None, "Only run if SciPy is installed")
+    def test_import_cython_function(self):
```

Comment:
Once the type of error to raise is determined, the correct raising of errors for modules that cannot be found and functions that are not present needs testing.